sabnzbd/downloader.py

```diff
@@ -691,7 +710,8 @@ def run(self):
                                     # Don't count this for the tries (max_art_tries) on this server
                                     self.__reset_nw(nw, count_article_try=False, send_quit=True)
                                     self.plan_server(server, _PENALTY_TOOMANY)
-                                    server.threads -= 1
+                                    if server.threads > 1:
+                                        server.threads -= 1
```

Comment:
Why is this needed? Can it go below 0? That sounds like a problem somewhere else.

Reply:
It's to stop it from going below 1, not below 0. It seems strange if 1 connection is too many.